examples/iotivity_examples/README.md

```diff
@@ -0,0 +1,314 @@
+server_oic: PUT request
+server_oic: key: state value: 0
+server_oic: LED0 is OFF
+LED_RED_OFF
+Outgoing message to [fe80:0000:0000:0000:a4ab:89ff:febd:1f80]:56789
+server_oic: GET request
+server_oic: Light state 0
+Outgoing message to [fe80:0000:0000:0000:a4ab:89ff:febd:1f80]:56789
+```
+TAPs interfaces can be easily deleted. Go to `/dist/tools/tapsetup` and type
+```
+$ sudo ./tapsetup -d
+```
+
+###<a name="sc_pput_samr21"></a> Server and Client (Periodic PUT) - SAMR21-XPRO target
+Now, we reproduce the previous scenario using two [SAMR21-XPRO][4] nodes.
+Connect your nodes, go to `/examples/iotivity-examples/server` and check the list of USB-connected nodes by typing:
+```
+$ make list-ttys
+```
+The output will be similar to
+```
+/sys/bus/usb/devices/2-1.3: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800001234', tty(s): ttyACM0
+/sys/bus/usb/devices/2-1.4: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800004321', tty(s): ttyACM1
+```
+We will use Serial Numbers in order to identify the designed node during the compilation phase.
+Now, we compile the server
+```
+$ make flash BOARD=samr21-xpro SERIAL=server_node_serial
+```
+then we open the serial connection
+```
+$ make term BOARD=samr21-xpro SERIAL=server_node_serial
+```
+The server starts and it is waiting for incoming requests.
+Now,  open a new terminal window, go to `/examples/iotivity-examples/client` and type
+```
+$ make flash BOARD=samr21-xpro SERIAL=client_node_serial
+$ make term BOARD=samr21-xpro SERIAL=client_node_serial
+```
+Client starts the discovery phase. Once it find a resource (with ResourceType **oic.r.light**), it registers as an observer on the resource, then it switches on its LED  and it finally starts with periodic PUT requests. The server LED will blink periodically.
+Client and Server terminal outputs are similar to the outputs in case of native target.
+###<a name="sc_sw_samr21"></a> Server and Client_Switch  - SAMR21-XPRO target
+This deployment emulates a smart home scenario in which we have a SmartBulb (server) and a SmartSwitch (client_switch). It requires two SAMR21-XPRO nodes or similar.
+Connect your nodes, go to `/examples/iotivity-examples/server` and check the list of USB-connected nodes by typing:
+```
+$ make list-ttys
+```
+The output will be similar to
+```
+/sys/bus/usb/devices/2-1.3: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800001234', tty(s): ttyACM0
+/sys/bus/usb/devices/2-1.4: Atmel Corp. EDBG CMSIS-DAP serial: 'ATML2127031800004321', tty(s): ttyACM1
+```
+We will use Serial Numbers in order to identify the designed node during the compilation phase.
+Now, we compile the server
+```
+$ make flash BOARD=samr21-xpro SERIAL=server_node_serial
+```
+then we open the serial connection
+```
+$ make term BOARD=samr21-xpro SERIAL=server_node_serial
+```
+The server starts and it is waiting for incoming requests.
+Now, we open a new terminal window, go to `/examples/iotivity-examples/client_switch` and type
+```
```

Comment:
Here, my .md editor/visualiser (MacDown) needs a new line, else it does not interpret it right (might be MacDown).